experimental/packages/exporter-trace-otlp-http/test/browser/CollectorTraceExporter.test.ts

```diff
@@ -432,6 +568,66 @@ describe('OTLPTraceExporter - web', () => {
         });
       });
     });
+
+    describe('when both "sendBeacon" and "XMLHttpRequest" are NOT available', () => {
+      let clock: sinon.SinonFakeTimers;
+      let stubXhr: sinon.SinonStub;
+      beforeEach(() => {
+        clock = sinon.useFakeTimers();
+
+        stubBeacon.value(undefined);
+        stubXhr = sinon.stub(globalThis, 'XMLHttpRequest').value(undefined);
+        fetchMock.mock('*', 200);
+        collectorTraceExporter = new OTLPTraceExporter(collectorExporterConfig);
+      });
+      afterEach(() => {
+        sinon.restore();
+        clock.restore();
+        fetchMock.restore();
+      });
+
+      const assertRequesetHeaders = (
```

Comment:
NOTE: accepting this in GitHub will break later uses of it. :slightly_smiling_face: 
```suggestion
      const assertRequestHeaders = (
```

Reply:
Thank you, fixed with 1dda1067.